tasks/task452_emea_bg_el_classification.json

```diff
@@ -0,0 +1,1061 @@
+{
+    "Contributors": [
+        "Vijay Kumawat"
+    ],
+    "Source": [
+        "emea bg-el (dataset link- https://huggingface.co/datasets/emea)"
+    ],
+    "Categories": [
+        "Classification"
+    ],
+    "Input_language": "Bulgarian",
+    "Output_language": "Greek",
+    "instruction_language": "English",
+    "Definition": "In this task, you are given a sentence in the Bulgarian language and corresponding Greek translation of this sentence. Here, your job is to generate label \"True\" if translation is correct, otherwise generate label \"False\".",
+
+    "Positive Examples": [
+        {
+            "input": "Поддържащата доза е 15 mg веднъж дневно, но при някои пациенти по- високите дози може да са от полза.",
+            "output": "True",
+            "explanation": "Translation of French sentence to Slovak sentence is correctly translated."
+        },
+        {
+            "input": "10 mgΌταν χρησιμοποιούνται ασθενείς αναστολείς του CYP3A4 (π. χ. διλτιαζέμη ή εσιταλοπράμη) ή του CYP2D6 ταυτόχρονα με το ABILIFY, είναι πιθανώς αναμενόμενες μικρές αυξήσεις των συγκεντρώσεων της αριπιπραζόλης.",
+            "output": "False",
+            "explanation": "French sentence is converted to Slovak sentence is incorrect.Hence it is classified as False."
+        }
+        
+    ],
+    "Negative Examples": [
+        {
+            "input": "9 Dans trois études court terme (4 à 6 semaines) contrôlées versus placebo portant sur 1.228 patients schizophrènes présentant des symptômes positifs ou négatifs, l'aripiprazole a montré une amélioration significativement plus importante des symptômes psychotiques comparativement au placebo.",
+            "output": "V troch krátkodobých (4 až 6 týždňových) placebom kontrolovaných štúdiách zahŕňajúcich 1 228 schizofrenických pacientov s pozitívnymi alebo negatívnymi symptómami sa aripiprazol spájal so štatisticky významnými väčšími zlepšeniami v psychotických symptómoch v porovnaní s placebom.",
+            "explanation": "Translation has missed digit 9 from the input."
```

Comment:
copy paste error. Ouptut should be True / False

Reply:
any comment on this one @vijaykumawat256 ?

Reply:
I have changed its output also. @kuntalkumarpal